apps/organization/repositories/organization.repository.ts

```diff
@@ -734,4 +734,84 @@ export class OrganizationRepository {
     }
   }
 
+  // eslint-disable-next-line camelcase
+  async getAllOrganizationDid(orgId: string): Promise<DidList[]> {
+    try {
+      return this.prisma.org_dids.findMany({
+        where:{
+          orgId
+        },
+        select:{
+          id: true,
+          createDateTime: true,
+          did: true,
+          lastChangedDateTime: true,
+          isPrimaryDid: true  
+        }
+      });
+    } catch (error) {
+      this.logger.error(`error in getAllOrganizationDid: ${JSON.stringify(error)}`);
+      throw new InternalServerErrorException(error);
```

Comment:
"Updated error handling to ensure compatibility with RpcException"
```suggestion
      throw error;
```